packages/atlas-experiment-table/webpack.config.js

```diff
@@ -57,8 +57,8 @@ module.exports = {
   devServer: {
     port: 9000,
     contentBase: path.resolve(__dirname, `html`),
-    publicPath: commonPublicPath
+    publicPath: commonPublicPath,
     // Add if developing a SPA to redirect non-matching routes known by WDS (i.e. no document in /html) to the router
-    // historyApiFallback: true
+    historyApiFallback: true
```

Comment:
Please also add a text exception for `TableManagerSpa.js`.

Reply:
What do you mean?

Reply:
Since we do not have test for TableManagerSpa.js , so we could add an entry in config to say that this file is out of tests.